main-app/app/scripts/bingoTicket/bingoModel.js

```diff
@@ -20,14 +20,29 @@
                 }
             };
 
+            var BingoCell = function () {
+                this.bingoNumber = null;
+                this.marked = "unmarked";
+                this.counted = false;
```

Comment:
This should be a function  - returns true if this.bingoNumber is null, or called. Or the opposite of that - depends what you mean by "counted". Also possible disambiguate the name?